core/xml.js

```diff
@@ -301,6 +301,25 @@ Blockly.Xml.domToWorkspace = function(xml, workspace) {
   if (workspace.setResizesEnabled) {
     workspace.setResizesEnabled(false);
   }
+  
+  // find last block on workspace to add new blocks at the bottom 
+  var blocks = workspace.getAllBlocks();
+  var bottomY =0; // container for the y of the bottom block
+  // sort the blocks in reverse order
+  if (blocks.length > 1) {
+	 blocks.sort(function(a, b) { // sort the block acording to their position and height
```

Comment:
Doing the change, I was wondering if, in RTL mode, the block position was only mirrored before to be added or if the block position was already reversed in the xml file. I assumed the first option, based on the existing code. This has importance for the getRelativeToSurfaceXY which won't give me the 'x' I look for otherwise (I would have to look for x + width)